src/nuget/Microsoft.Build.Tasks.Core.nuspec

```diff
@@ -19,8 +19,9 @@
       <group targetFramework="dotnet">
         <dependency id="Microsoft.Build.Framework" />
         <dependency id="System.IO.FileSystem.DriveInfo" version="4.0.0-beta-23302" />
-        <dependency id="Microsoft.Win32.Primitives" version="4.0.1-beta-23516" />
         <dependency id="Microsoft.Win32.Registry" version="4.0.0-beta-23516" />
```

Comment:
Shouldn't you also just remove Microsoft.Win32.Registry dependency from these Core nuspecs? That will help ensure no one accidentally depends on it.